working/m12-strings-english.txt

```diff
@@ -2096,53 +2387,64 @@
 ^L11003^[00 FF]
 ^L11004^[8D FF 02 00][A8 FF 00] and [90 FF 00][1A FF 01 00][01 FF] [00 FF]
 ^L11005^ and [8D FF 01 00][9A FF 02 00][81 FF _L11006_]his friends[01 FF] [00 FF]
 ^L11006^her friends[01 FF] [00 FF]
-^L11007^[BC FF 01 00]^L11008^[C0 FF][A8 FF 01][96 FF 00][81 FF _L11009_][8D FF 00 00][92 FF 00 01 02][81 FF _L11009_][8D FF 00 00][92 FF 00 01 03][81 FF _L11009_][80 FF _L11008_]
+^L11007^[BC FF 01 00]
+^L11008^[C0 FF][A8 FF 01][96 FF 00][81 FF _L11009_][8D FF 00 00][92 FF 00 01 02][81 FF _L11009_][8D FF 00 00][92 FF 00 01 03][81 FF _L11009_][80 FF _L11008_]
 ^L11009^[A8 FF 01][87 FF][95 FF 05 _L11010_ _L11010_ _L11011_ _L11011_ _L11011_][00 FF]
 ^L11010^[00 FF]
 ^L11011^ each[00 FF]
 [C2 FF][1A FF 05 00][86 FF _L3623_][00 FF]
-^L2109^[68 FF][C4 FF 00 05][01 FF][5F FF 30][21 FF][1B FF 78 00]^L3625^[FC FF 11 00][81 FF _L3622_]^L3624^[02 FF]@[86 FF _L11000_] gained [AC FF] exp[86 FF _L11007_].[22 FF][00 FF]
+^L2109^[68 FF][C4 FF 00 05][01 FF][5F FF 30][21 FF][1B FF 78 00]
+^L3625^[FC FF 11 00][81 FF _L3622_]
+^L3624^[02 FF]@[86 FF _L11000_] gained [AC FF] exp[86 FF _L11007_].[22 FF][00 FF]
 ^L2110^[68 FF][C4 FF 00 B8][01 FF][5F FF 30][21 FF][1B FF 78 00][80 FF _L3625_]
-^L2111^[68 FF][01 FF][5F FF 30][21 FF][1B FF 5A 00]@[86 FF _L11000_] gained [AC FF] exp[22 FF][00 FF]
-^L2112^[1B FF 0A 00][C4 FF 00 07] lost the battle...[22 FF][00 FF]
+^L2111^[68 FF][01 FF][5F FF 30][21 FF][1B FF 5A 00][01 FF]@[86 FF _L11000_] gained [AC FF] exp[86 FF _L11007_].[22 FF][00 FF]
+^L2112^[1B FF 0A 00][C4 FF 00 07]@[90 FF 00][1A FF 01 00] lost the battle...[22 FF][00 FF]
 ^L2113^@[AD FF]'s level is[01 FF]  now [AC FF]![22 FF][00 FF]
 ^L2114^@[86 FF _L3626_]Offense went up by [AC FF]![22 FF][00 FF]
-^L2115^[86 FF _L3626_]Defense went up by [AC FF]![22 FF][00 FF]
-^L2116^[86 FF _L3626_]Speed went up by [AC FF]![22 FF][00 FF]
-^L2117^[86 FF _L3626_]Guts went up by [AC FF]![22 FF][00 FF]
-^L2118^[86 FF _L3626_]Vitality went up by [AC FF]![22 FF][00 FF]
-^L2119^[86 FF _L3626_]IQ went up by [AC FF]![22 FF][00 FF]
-^L2120^[86 FF _L3626_]Luck went up by [AC FF]![22 FF][00 FF]
-^L2121^[86 FF _L3627_]Maximum HP went up by [AC FF]![22 FF][00 FF]
-^L2122^[86 FF _L3628_]Maximum PP went up by [AC FF]![22 FF][00 FF]
-^L2123^[D4 FF 67 00][C1 FF][87 FF]@[AD FF] realized the power of[01 FF]  [2D FF].[22 FF][00 FF]
+^L2115^@[86 FF _L3626_]Defense went up by [AC FF]![22 FF][00 FF]
+^L2116^@[86 FF _L3626_]Speed went up by [AC FF]![22 FF][00 FF]
+^L2117^@[86 FF _L3626_]Guts went up by [AC FF]![22 FF][00 FF]
+^L2118^@[86 FF _L3626_]Vitality went up by [AC FF]![22 FF][00 FF]
+^L2119^@[86 FF _L3626_]IQ went up by [AC FF]![22 FF][00 FF]
+^L2120^@[86 FF _L3626_]Luck went up by [AC FF]![22 FF][00 FF]
+^L2121^@[86 FF _L3627_]Maximum HP went up by [AC FF]![22 FF][00 FF]
+^L2122^@[86 FF _L3628_]Maximum PP went up by [AC FF]![22 FF][00 FF]
+^L2123^[D4 FF 67 00][C1 FF][87 FF]@[AD FF] learned the power of[01 FF]  [2D FF].[22 FF][00 FF]

```

Comment:
The original game used the text "realized" here and in a few other places, so you'll either want to be consistent with them or change them to consistent with this.